docs/source/actions/draft.rst

```diff
@@ -0,0 +1,32 @@
+.. meta::
+   :description: Mergify Documentation for Draft action
+   :keywords: mergify, draft, pull request
+   :summary: Convert pull requests to Draft. Useful to avoid pinging reviewers too early.
+   :doc:icon: trash
+
+.. _draft action:
+
+draft
+=====
+
+The ``draft`` action converts the pull request to a draft, avoiding 
+this way to ping reviewers until the pull request is ready to merge.
+
+Examples
+--------
+
+📜 Converting Pull-Request to Draft
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You might want to work on your Pull-Request a bit more until it is
+ready for a review. To avoid automatically pinging reviewers before
+it is ready to merge, you can convert it to draft using
+the ``draft`` action by writing a rule such as:
+
+.. code-block:: yaml
+
+    pull_request_rules:
+    - name: convert to draft
+      conditions:
+        actions:
+          draft:
```

Comment:
```suggestion
If any of your continuous integration checks fails, it might be worth converting the pull request to a draft automatically since it's likely not ready to review.

.. code-block:: yaml

    pull_request_rules:
    - name: convert to draft
      conditions:
        - "#check-failed>0"
      actions:
        draft:
```